velox/vector/tests/VectorMaker.cpp

```diff
@@ -220,6 +220,39 @@ ArrayVectorPtr VectorMaker::arrayOfRowVector(
       0);
 }
 
+ArrayVectorPtr VectorMaker::createArrayVector(
+    const std::vector<vector_size_t>& offsets,
+    const std::vector<vector_size_t>& sizes,
+    const VectorPtr elementVector,
+    const std::vector<vector_size_t>& nulls) {
+  BufferPtr offsetsBuffer = allocateOffsets(offsets.size(), pool_);
+  BufferPtr sizesBuffer = allocateSizes(sizes.size(), pool_);
+  BufferPtr nullsBuffer =
+      AlignedBuffer::allocate<bool>(sizes.size(), pool_, bits::kNotNull);
+  auto rawOffsets = offsetsBuffer->asMutable<vector_size_t>();
+  auto rawSizes = sizesBuffer->asMutable<vector_size_t>();
+  auto rawNulls = nullsBuffer->asMutable<uint64_t>();
+
+  memcpy(rawOffsets, offsets.data(), offsets.size() * sizeof(vector_size_t));
+  memcpy(rawSizes, sizes.data(), sizes.size() * sizeof(vector_size_t));
+
+  if (!nulls.empty()) {
+    for (int i = 0; i < nulls.size(); i++) {
+      VELOX_CHECK_EQ(sizes[nulls[i]], 0)
+      bits::setNull(rawNulls, nulls[i]);
```

Comment:
I wonder how convenient it is to specify nulls as a list of indices.

Reply:
Since when testing generally only a few nulls are present , I find this more convenient.